docs/02-get-started/05-observability.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Observability
+---
+
+We successfully got our Function [triggered](04-trigger-workload-with-event.md).
+Now we would like to check its logs. 
+To do that, we'll use the Grafana dashboard that comes with Kyma. 
+
+> **NOTE:** See how to access logs from the Function's Pod [via Kyma Dashboard](../04-operation-guides/operations/obsv-01-access-logs.md#kubernetes-logs-in-kyma-dashboard) and [using kubectl](../04-operation-guides/operations/obsv-01-access-logs.md#kubernetes-logs-using-kubectl). 
+
```

Comment:
I'd prefer if we didn't duplicate port forwarding instructions here and in the "Access and Expose..." guide. But it makes sense that we want to provide all required instructions in the GSG and not point the user to another tutorial. Maybe we can add some info like

```suggestion
In this tutorial, we'll access Grafana with port forwarding. You can also [expose the service securely](../04-operation-guides/operations/obsv-02-access-expose-kiali-grafana/#kubernetes-logs-in-kyma-dashboard) for easier access in the future.
```
That is, unless the navigation links in Busola aren't going to work...

Reply:
I feel you, but at the sama time I don't think context switching is a good thing, and I'd really prefer to have all the code in one place. I'd link only to more detailed explanation of things. 
Plus it's not like this is something we'll have to maintain anyway (as in: the command and the port are HIGHLY unlikely to change, and even if they do, we are supposed to test the tutorials every release anyway).

Reply:
I'll add the link. Thanks! :)

Reply:
I agree, the commands hopefully are very stable :)